### PR TITLE
[CON-362] Implement google analytics tracking scripts

### PR DESCRIPTION
**Pull request by cofaulco**
Formatting makes it look like a lot more changes than there are

### CODE REVIEW

**Comment by lazcool** [comment]
> Formatting makes it look like a lot more changes than there are

I'll tick `Hide whitespace changes` 😉

File: src/SFA.DAS.Support.Portal.Web/Views/Shared/_GoogleAnalyticsHeadPartial.cshtml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<!-- Google Tag Manager -->
+
+<script>
```
Comment (lazcool) [inline src/SFA.DAS.Support.Portal.Web/Views/Shared/_GoogleAnalyticsHeadPartial.cshtml]: That's the neatest version of this script I've ever seen 😄. We could use the more minified version from [here](https://developers.google.com/tag-manager/quickstart), but it's all present and correct!

File: src/SFA.DAS.Support.Portal.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -35,36 +35,37 @@
     <meta name="theme-color" content="#0b0c0c"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta property="og:image" content="@Url.Content("~/assets/images/opengraph-image.png")">
-    @Html.Partial("_Analytics")
+    @Html.Partial("_GoogleAnalyticsHeadPartial")
```
Comment (lazcool) [inline src/SFA.DAS.Support.Portal.Web/Views/Shared/_Layout.cshtml]: The recommendation is to place this as close to the opening `<head>` tag as possible.

Comment (cofaulco) [reply src/SFA.DAS.Support.Portal.Web/Views/Shared/_Layout.cshtml]: I agree, I just placed it over what was already there which was some random script that I don't think was working. Will move it up

Comment (cofaulco) [reply src/SFA.DAS.Support.Portal.Web/Views/Shared/_Layout.cshtml]: moved this higher

**Pull request merged**